packages/table/preview/index.tsx

```diff
@@ -286,6 +286,7 @@ ReactDOM.render(
     getTableComponent(200, 100 * 1000, {} , {
         fillBodyWithGhostCells: true,
         selectionModes: SelectionModes.ALL,
+        isRowResizable: true,
```

Comment:
Do we want to keep this? (It was helpful for this PR.) Or do we want to wait until we build one Table example to rule them all.